src/lib/core/core.gni

```diff
@@ -47,6 +47,9 @@ declare_args() {
   # Configure chip logging to output through pigweed logging.
   chip_use_pw_logging = false
 
+  # Configure chip logging to output through external logging implementation.
```

Comment:
Some documentation on what 'external logging' means would be useful. Like making sure that something implements `void ENFORCE_FORMAT(3, 0) LogV(const char * module, uint8_t category, const char * msg, va_list v)` (is this what is needed?)

Reply:
Yes, the goal is to allow external code to provide LogV implementation. I will update the comment to be more explicit. Thanks,